rmw_dds_common/test/test_graph_cache.cpp

```diff
@@ -891,19 +891,19 @@ TEST(test_graph_cache, normal_usage)
     graph_cache,
   {
     // topic1
-    {"reader6", "remote_participant", "topic1", "Str", true},
-    {"reader7", "remote_participant", "topic1", "Custom", true},
+    {"reader6", "remote_party", "topic1", "Str", true},
+    {"reader7", "remote_party", "topic1", "Custom", true},
```

Comment:
Why do we have to change this name?

Reply:
The 16 byte uids cannot accomodate names like "remote_participant" which is 18 bytes long. The closest short alternative I could think of, "remote_party", fits in under 16 bytes.